data/Symbeeosis/codedLists.ttl

```diff
@@ -0,0 +1,55 @@
+bdg:AssayCompound a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Assay compound codelist class".
+<assayCompound> a skos:ConceptScheme;
+  rdfs:label "Assay compound codelist scheme".
+<assayCompound/DPPH> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
+  skos:prefLabel     "DPPH" ;
+  skos:exactMatch       wd:Q5205953 .
+<assayCompound/ABTS> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "ABTS" ;
+  skos:exactMatch       wd:Q287582 .
+<asseyCompound/Trolox> a skos:Concept, bdg:AssayCompound ;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Trolox" ;
+  skos:exactMatch       wd:Q245489 .
+<asseyCompound/GallicAcid> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Gallic Acid" ;
+  skos:exactMatch       wd:Q375837 .
+<asseyCompound/Quercetin> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <asseyCompound>; skos:topConceptOf <asseyCompound>;
+  skos:prefLabel     "Quercetin" ;
+  skos:exactMatch       wd:Q409478 .
+
+<extractionMethod> a skos:ConceptScheme;
```

Comment:
```suggestion
<preparationMethod> a skos:ConceptScheme;
```
These methods just break up the leaves, they don't extract any substance from the leaves